tests.py

```diff
@@ -1,7 +1,12 @@
 from storage import Storage
 
-def test_add():
-    pass
+def test_add(): #Катя
+    st = Storage({'a': 1, 'b': 2})
+    key = 'ItsAKEy'
+    value = 'ItsAValue'
+    
+    st.add(key, value)
+    print(st)
```

Comment:
Assert statement is expected here instead of print one. For example:
`assert st.data[key] == value, "The key {} is absent or value is not equal to expected {}".format(key, value)`

Reply:
fixed it 👍

Reply:
If you respect print, it's a good idea to reimplement _repr()_ method in class Storage. It should return a printable representation of the given object. 
If it is not reimplemented, you get smth like this:
```
>>> st = Storage({"one":1, "two":2})
>>> print(st)
>>> <storage.Storage object at 0x7f9ea8013150>
```